llvm/lib/Object/Minidump.cpp

```diff
@@ -154,3 +155,17 @@ MinidumpFile::create(MemoryBufferRef Source) {
   return std::unique_ptr<MinidumpFile>(
       new MinidumpFile(Source, Hdr, *ExpectedStreams, std::move(StreamMap)));
 }
+
+Expected<ArrayRef<MemoryDescriptor_64>> MinidumpFile::getMemory64List() const {
+  Expected<minidump::Memory64ListHeader> MemoryList64 = getMemoryList64Header();
+  if (!MemoryList64)
+    return MemoryList64.takeError();
+
+  std::optional<ArrayRef<uint8_t>> Stream =
+      getRawStream(StreamType::Memory64List);
+  if (!Stream)
+    return createError("No such stream");
```

Comment:
```suggestion
  ArrayRef<uint8_t> Stream =
      getRawStream(StreamType::Memory64List).value();
```
The error can never happen as the first call checks for stream existence.

I can also imagine a pattern like:
```
Stream = getRawStream(...);
Header = getDataSliceAs<Header>(Stream, 0, 1);
return getDataSliceAs<List>(Stream, sizeof(Header), Header->NumberOfMemoryRanges);
```

which might make it clearer that the header and body come from the same stream.

```